AK/StringView.cpp

```diff
@@ -7,15 +7,19 @@
 #include <AK/AnyOf.h>
```

Comment:
Typo in commit title: "StringVIew" -> "StringView"

Reply:
Thanks, fixed!